otterdog/eclipse-bluechi.jsonnet

```diff
@@ -165,5 +165,30 @@ orgs.newOrg('eclipse-bluechi') {
         },
       ],
     },
+    orgs.newRepo('ansible-bluechi') {
```

Comment:
Just wondering a name, wouldn't `bluechi-ansible-collection` be a better name?

Reply:
Sure, np. Done